_data/simple-icons.json

```
@@ -8462,6 +8462,11 @@
             "hex": "03234B",
             "source": "https://www.st.com/"
         },
+        {
+            "title": "StopStalk",
+            "hex": "536DFE",
+            "source": "https://github.com/stopstalk/media-resources/blob/265b728c26ba597b957e72134a3b49a10dc0c91d/stopstalk-small-black.svg"
```

As suggested by @service-paradis, we should specify the license [as seen in the media-resources repository](https://github.com/stopstalk/media-resources/blob/master/LICENSE).

```suggestion
            "title": "StopStalk",
            "hex": "536DFE",
            "source": "https://github.com/stopstalk/media-resources/blob/265b728c26ba597b957e72134a3b49a10dc0c91d/stopstalk-small-black.svg",
            "license": {
                "type": "MIT"
            }
```